internal/services/containers/kubernetes_cluster_other_resource_test.go

```diff
@@ -423,6 +423,32 @@ func TestAccKubernetesCluster_windowsProfile(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesCluster_windowsProfileGMSA(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_kubernetes_cluster", "test")
+	r := KubernetesClusterResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.windowsProfileGMSAEmptyPropertyConfig(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("kube_config.0.client_key").Exists(),
```

Comment:
The `check.That(data.ResourceName).Key("xxx").Exists(),` is not necessary, because in the below `import step`, it will create an empty terraform workspace and import this resource, then compare the state between two workspaces, so it will be checked.

Reply:
Updated